lib/vagrant/plugin/v2/trigger.rb

```diff
@@ -30,58 +30,97 @@ def initialize(env, config, machine, ui)
           @logger = Log4r::Logger.new("vagrant::trigger::#{self.class.to_s.downcase}")
         end
 
-        # Fires all triggers, if any are defined for the action and guest. Returns early
+        # Fires all triggers, if any are defined for the named type and guest. Returns early
         # and logs a warning if the community plugin `vagrant-triggers` is installed
         #
-        # @param [Symbol] action Vagrant command to fire trigger on
+        # @param [Symbol] name Name of `type` thing to fire trigger on
```

Comment:
Maybe we can be a bit more descriptive here instead of saying thing... :thinking: We could mention what a valid type here is perhaps.